llvm/test/CodeGen/RISCV/zcmp-additional-stack.ll

```diff
@@ -3,7 +3,7 @@
 define ptr @func(ptr %s, i32 %_c, ptr %incdec.ptr, i1 %0, i8 %conv14) #0 {
 ; RV32-LABEL: func:
 ; RV32:       # %bb.0: # %entry
-; RV32-NEXT:    cm.push {ra, s0-s1}, -24
+; RV32-NEXT:    cm.push {ra, s0-s1}, -32
 ; RV32-NEXT:    .cfi_def_cfa_offset 24
```

Comment:
Aren't these cfi directives wrong, since the stack pointer was moved by 32 not 24?